### ISSUE TITLE
Error messages in fields do not appear

### ISSUE THREAD

**Issue by marcemarin** [codenco-dev/nova-grid-system#12]
Hello, I have a question, when using your package, the error messages that used to appear below the fields have stopped appearing, are you aware? you can help?
![4p6h64Tv](https://user-images.githubusercontent.com/7118089/95336448-13c1b080-0887-11eb-9c56-0bb4b62fd067.jpg)

**Comment by thomasdominic** [comment]
Hi,

I have been tested and the error messages are displayed correctly.
Did you test your code while deactivating the package?
![Capture d’écran de 2020-10-08 05-39-03](https://user-images.githubusercontent.com/17202290/95412314-ffce8b00-0928-11eb-80ce-d533f6a3f2be.png)

**Comment by marcemarin** [comment]
Yes, I have tried it, it works fine without the package. Could it be a nova
version theme? I have the 2.0
